lua/wire/client/e2descriptions.lua

```diff
@@ -313,6 +313,13 @@ E2Helper.Descriptions["setTrails(e:nnnsvn)"] = "StartSize, EndSize, Length, Mate
 E2Helper.Descriptions["setTrails(e:nnnsvnnn)"] = "StartSize, EndSize, Length, Material, Color (RGB), Alpha, AttachmentID, Additive. Adds a trail to E with the specified attributes"
 E2Helper.Descriptions["removeTrails(e:)"] = "Removes the trail from E"
 E2Helper.Descriptions["runOnKeys(en)"] = "If set to 1, E2 will run when specified player presses/releases their key"
+E2Helper.Descriptions["disconnectClk()"] = "Returns 1 if the chip is being executed because of a disconnect event. Returns 0 otherwise"
+E2Helper.Descriptions["lastDisconnected()"] = "Returns the last player to disconnect. Must be done while in a disconnectClk() as anytime after the player object is gone."
+E2Helper.Descriptions["runOnDisconnect(n)"] = "If set to 0, the chip will no longer run on disconnect events, otherwise it makes this chip execute when someone disconnects. Only needs to be called once, not in every execution"
```

Comment:
> what are you even saying

To avoid repetition of myself, read my comment [below](https://github.com/wiremod/wire/pull/1222#issuecomment-250594815).